### DISCUSSION TITLE
Unable to upload to LotW

### DISCUSSION THREAD

**Discussion by amhyder** [wavelog/wavelog#532] [Q&A] [answered]
Hi everyone, I'm having some issues uploading to LotW. Now that the service is back up I can confirm it's not an issue of the LotW site being down.

I am using a docker installation of Cloudlog.

I have gone through the process of uploading my .p12 cert and added my login credentials to the config file. I have tried to use the "manual sync" button on the LotW 3rd party upload page but am having trouble. I can see a .tq8 file is being reported as created. I can also see that some confirmations were pulled in. The strage thing is, upon entering my docker container, I cannot find any files under the tq8 log file name. I used a find command starting at the root directory and came up empty.

I'm also unable to find any mention of my .p12 cert file, which leads me to believe that these files are not correctly being processed. I know that LotW is not getting my logs because the activity page does not update with a new entry. I can see it on the LotW upload screen and it's showing a status of 'valid'.

I have also turned on logging levels 1 and 2 in the config and am unable to find any errors reported by the manual sync action. Additionally, when manually syncing multiple times, the same 2 QSOs are listed every time I sync, which seems like the log entry is not being updated.

Can anyone assist?

**Comment by int2001** [comment, marked as answer]
GM @amhyder - lotw is under huge load since it is back online.
The lotw-process includes a queue (located at arrl). So process looks like the following:
- Wavelog signs the file with your cert
- Wavelog uploads it to lotw (and sets last-upload, green arrows and so on)
- lotw takes this file and puts it and the END of their backlog --> https://www.arrl.org/logbook-queue-status
- if the queue reaches your file it'll process it.

This causes that your QSOs are not instantly marked as received (at lotw).
While having a look at the queue (see link above) i recognize that there's still a "gap" of 1h. So hopefully your logs are now processed there.

HTH

**Reply by amhyder** [reply]
Thanks for pointing this out, it seems that overnight my activity page has updated with the expected information. 
Out of curiosity, are the `.tq8` files transient and not saved to the file system? I was expecting to find them in the upload directory but was unsure if there's a cleanup process that removes them upon upload.

Thanks again for you and the team's hard work on Wavelog! It's been a joy to use and I'm looking forward to future development!

**Reply by int2001** [reply]
Exactly. They're only produced temporary for upload and deleted immediatly afterwards.
